src/group_theory/free_group.lean

```diff
@@ -863,4 +883,66 @@ fintype.subtype (list.to_finset $ red.enum L₁) $
 
 end reduce
 
+section metric
+
+variable [decidable_eq α]
+
+/-- The length of reduced words provides a norm on a free group. --/
+def norm (x : free_group α) : nat := x.to_word.length
+
+private lemma norm_inv_le (x : free_group α) : norm x⁻¹ ≤ norm x :=
+begin
+  let w := x.to_word,
+ 
+  calc norm x⁻¹ = norm ((mk (x.to_word))⁻¹) : by simp only [to_word.mk]
+       ... = norm (mk (inv_rev w)) : by simp only [←inv_mk]
+       ... ≤ (inv_rev w).length : by exact to_word.le_length
+       ... = w.length : inv_rev_length
+       ... = norm x : by unfold norm,
```

Comment:
```suggestion
       ... = norm x : rfl
```
In general, if something is provable by `unfold`, it's probably provable by `rfl` (or `by refl`).